lms/templates/main.html

```diff
@@ -39,12 +39,6 @@
       </script>
   % endif
 
-  <%
-    jsi18n_path = "js/i18n/{language}/djangojs.js".format(language=LANGUAGE_CODE)
```

Comment:
Answer seems to be: no.  It's a file with global functions tied to static maps.  It only seem to get called by JS that is rendering Underscore templates.  So, as long as it loads before that, it'll be available for any rendering and won't flash.